controller/twitter.js

```diff
@@ -1,21 +1,62 @@
 'use strict';
 
-let Twitter = require('twitter'),
+let Writable = require('stream').Writable,
+    TwitterStream = require('twitter-stream-api'),
     config = require('../config/config');
 
```

Comment:
the new node acts in the same way as the old one did, except it takes into account the real time tweets as well